extensions/MikeDEV/peerjs.js

```diff
@@ -0,0 +1,929 @@
+        means.
+
+        In jurisdictions that recognize copyright laws, the author or authors
+        of this software dedicate any and all copyright interest in the
+        software to the public domain. We make this dedication for the benefit
+        of the public at large and to the detriment of our heirs and
+        successors. We intend this dedication to be an overt act of
+        relinquishment in perpetuity of all present and future rights to this
+        software under copyright law.
+
+        THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+        EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+        MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+        IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+        OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+        ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+        OTHER DEALINGS IN THE SOFTWARE.
+
+        For more information, please refer to http://unlicense.org/
+    */
+
+  /* eslint-disable */
+  // prettier-ignore
+  function ULID(){let $=["0","1","2","3","4","5","6","7","8","9","A","B","C","D","E","F","G","H","J","K","M","N","P","Q","R","S","T","V","W","X","Y","Z"],e=-1,f=new Uint8Array(16),_=new DataView(f.buffer,0,6),n=new Uint8Array(f.buffer,6,10),t=Array(26);return function(){let r=Date.now();if(r===e)for(let o=9;o>=0&&!(n[o]++<255);o--);else e=r,_.setUint16(0,r/4294967296|0),_.setUint32(2,0|r),window.crypto.getRandomValues(n);return function e(f){t[0]=$[f[0]>>5],t[1]=$[f[0]>>0&31];for(let _=0;_<3;_++)t[8*_+2]=$[f[5*_+1]>>3],t[8*_+3]=$[(f[5*_+1]<<2|f[5*_+2]>>6)&31],t[8*_+4]=$[f[5*_+2]>>1&31],t[8*_+5]=$[(f[5*_+2]<<4|f[5*_+3]>>4)&31],t[8*_+6]=$[(f[5*_+3]<<1|f[5*_+4]>>7)&31],t[8*_+7]=$[f[5*_+4]>>2&31],t[8*_+8]=$[(f[5*_+4]<<3|f[5*_+5]>>5)&31],t[8*_+9]=$[f[5*_+5]>>0&31];return t.join("")}(f)}}
+
+  class PeerJS_Scratch {
+    constructor() {
+      this.peer;
+      this.dataConnections = new Map();
+      this.voiceConnections = new Map();
+      this.newestConnected = "";
+      this.lastDisconnected = "";
+      this.hasMicPerms = false;
+      this.verboseLogs = false;
+      this.ringingPeers = new Map();
+      this.ulidGenerator = new ULID();
+      this.myVoiceStream;
+    }
+
+    getInfo() {
+      return {
+        id: "mikedevpeerjs",
+        name: "PeerJS",
+        blockIconURI: blockIconURI,
+        menuIconURI: menuIconURI,
+        color1: "#E96151",
+        blocks: [
+          {
+            opcode: "newULID",
+            blockType: Scratch.BlockType.REPORTER,
+            disableMonitor: true,
+            text: Scratch.translate("generate ULID"),
+          },
+          "---",
+          {
+            opcode: "enableVerboseLogs",
+            blockType: Scratch.BlockType.COMMAND,
+            text: Scratch.translate(
+              "enable verbose browser console logs? [VERBOSE]"
```

Comment:
Maybe it could be
```js
"[TOGGLE] verbose browser console logs"
```
`TOGGLE` would be an `enable/disable` menu

Reply:
Will make the requested changes as a new block and hide this since I have a demo that uses this opcode.